spec/helpers/feedback_messages_helper_spec.rb

```diff
@@ -10,6 +10,15 @@
     end
   end
 
+  describe "#reporter_email_details" do
+    it "have proper subject and body" do
```

Comment:
```suggestion
    it "has proper subject and body" do
```
The world's tiniest nitpick but I think we want this to read as "it has the proper subject and body" :)

Reply:
Oops! I copied and pasted this without even noticing... great catch!! 😊